package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensor-hq/tensor-common",
-  "version": "5.1.0",
+  "version": "5.1.1",
```

Comment:
Not sure if this would be a 5.2.0 or 5.1.2
This is jiust bringing back old functions that have been deprecated